### PR TITLE
#158518429 update driver profile 

### PR DESCRIPTION
**Pull request by artorious**
#### What does this PR do?

Adds drivers' profile update form for user to change;
 
* Password 
* Email address
* Username
* Liscence No.

#### Description of Task to be completed?

* Building (without functionality), the UI elements with HTML&CSS. 

#### How the app should be tested
On a webbrowser:

* [Download](https://github.com/artorious/Ride-My-Way/archive/ft-driver-profile-update-158518429.zip) the repo
* Extract and navigate into the `Ride-My-Way` folder
* in the `UI` folder, open `update_driver_profile.html` on a browser

On a  terminal (Linux):

* Install and set up git `sudo apt install git` 

* Clone the repository `git clone https://github.com/artorious/Ride-My-Way.git`

* Run `git checkout ft-driver-profile-update-158518429`

* Run `cd Ride-My-Way/UI/`

* Open `update_driver_profile.html` on a browser

#### Any background context you want to provide?
The UI has no functionality. 
Developed using HTML and CSS

#### What are the relevant pivotal tracker stories?
[#158518429](https://www.pivotaltracker.com/story/show/158518429)

#### Screenshots 

driverupdate

#### Questions:

### CODE REVIEW

**Comment by meshack-mbuvi** [comment]
It looks good

**Comment by LWanjiru** [comment]
Looks good, however, the two buttons at the bottom are too close to each other, please add space between them

File: UI/update_driver_profile.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          <input type="submit" value="Update">       
+        </div>
+      </fieldset>
+    </form>
+  </div>
+  <hr>
+  <footer>&copy;RideMyWay 2018.<br><a href="#">Contact Us.</a></footer>
+</div>
+</body>
+</html>
```
Comment (LWanjiru) [inline UI/update_driver_profile.html]: add new line

**Review by LWanjiru — CHANGES_REQUESTED** [review]

**Comment by artorious** [comment]
@meshack-mbuvi @LWanjiru Your input has been invaluable. Thanks
drvupd8t

**Pull request merged**